src/main/java/ejercicio/banco/repository/CsvAccountRepository.java

```diff
@@ -24,54 +24,36 @@ public CsvAccountRepository() {
 
     @Override
     public List<Account> findAll(String filename) {
-
```

Comment:
In general, is a good practice to **always** format the code before pushing (search on the internet how to format the code in IntelliJ).